bin/pot-to-php.js

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env node
```

Comment:
If we drop this and instead call from the npm script as `node ./bin/pot-to-php.js`, wouldn't we have the advantage of added Windows support?

Reply:
Sorry, for some reason email notifications for code reviews were turned off.

I can definitely see this change. I copied it from `./bin/create-php-parser.js`. After this is merged I can create a PR that addresses both files.